### PR TITLE
Es8 trailing comma

### PR DESCRIPTION
**Pull request by jugglinmike**
This is intended to resolve gh-3265

### CODE REVIEW

File: tests/test262/expectations.txt [hunk 1]
```diff
@@ -7208,42 +7142,18 @@ test/language/statements/class/gen-meth-dflt-params-ref-later.js(default)
...
 test/language/statements/class/meth-static-dflt-params-ref-later.js(default)
 test/language/statements/class/meth-static-dflt-params-ref-later.js(strict mode)
 test/language/statements/class/meth-static-dflt-params-ref-self.js(default)
 test/language/statements/class/meth-static-dflt-params-ref-self.js(strict mode)
-test/language/statements/class/meth-static-dflt-params-trailing-comma.js(default)
-test/language/statements/class/meth-static-dflt-params-trailing-comma.js(strict mode)
-test/language/statements/class/meth-static-params-trailing-comma-multiple.js(default)
-test/language/statements/class/meth-static-params-trailing-comma-multiple.js(strict mode)
-test/language/statements/class/meth-static-params-trailing-comma-single.js(default)
-test/language/statements/class/meth-static-params-trailing-comma-single.js(strict mode)
```
Comment (rwaldron) [inline tests/test262/expectations.txt]: Damn, this is so awesome

**Comment by rwaldron** [comment]
@jugglinmike please review and resolve conflicts. Ping me when ready for follow up

**Comment by jugglinmike** [comment]
Sure thing. Original version of this branch is available here:

https://github.com/jshint/jshint/compare/master...jugglinmike:es8-trailing-comma-prebase

**Comment by rwaldron** [comment]
Thanks again!

**Pull request merged**